src/config.rs

```diff
@@ -178,6 +186,10 @@ impl Default for TermvizConfig {
                 topic: "pointcloud2".to_string(),
                 use_rgb: false,
             }],
+            polygon_stamped_topics: vec![ListenerConfigColor {
```

Comment:
there is https://github.com/carzum/termviz/blob/main/src/footprint.rs already. The footprint will be drawn twice now.   We could drop the old module for the more generic polygon stamped approach?

Reply:
I did a quick prototype of this (see https://github.com/ValerioMa/termviz/pull/1/files ). The change is not huge but it is changing some logic and IMO it is worth do discuss in a separate PR.